services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -180,6 +180,10 @@ def _can_save_is_no_longer_none(cls, values):
             values["can_save"] = False
         return values
 
+    @classmethod
+    def make_default(cls) -> "ServiceRemovalState":
+        return cls(can_remove=False, can_save=False, was_removed=False)
```

Comment:
this is a bit strange ... why no using the pydantic default mechanism ?
it is confusing to have no defaults in the definition and an extra function to make defaults?

```python
class ServiceRemovalState(BaseModel):
    can_remove: bool = False
    can_save: bool = False
    was_removed: bool = False

class DynamicSidecar(BaseModel):
         service_removal_state: ServiceRemovalState = Field(default = ServiceRemovalState() )
        # ...

```

Reply:
agree

Reply:
Dropping this